### PR TITLE
Revised @System.CopySecurityOptions help page, both English and Russian.

### PR DESCRIPTION
**Pull request by MKadaner**

### CODE REVIEW

**Comment by MKadaner** [comment]
Please review. I'll update other files as necessary when the change is ready.

File: far/FarEng.hlf.m4 [hunk 1]
```diff
@@ -6121,43 +6121,56 @@ files, like #Temporary panel# does.
...
+
+ Examples:
+
+ #0b100'001# - ^<wrap>In the Move dialog, the #Access rights# option
+is always set to #Copy#.
+             In the Copy dialog, the option is initially set to
+#Default#; then the previous user’s choice is remembered (within the
+current Far session).
+
+ #0b011'100# - ^<wrap>In the Move dialog, the #Access rights# option
```
Comment (alabuzhev) [inline far/FarEng.hlf.m4]: I like the binary notation, but it's not what the user will see in the config editor or an exported XML.
Maybe something like `011100 (28)`?

File: far/FarEng.hlf.m4 [hunk 2]
```diff
@@ -6121,43 +6121,56 @@ files, like #Temporary panel# does.
...
+ Bit 0    Bit 3    0 - ^<wrap>#Default# (bits 1 / 4 are ignored)
+                   1 – controlled by bits 1 / 4
 
- Примечание:
+ Bit 1    Bit 4    0 - ^<wrap>#Copy# (if bit 0 / 3 is set to 1)
+                   1 - #Inherit# (if bit 0 / 3 is set to 1)
 
- Параметр "System.CopySecurityOptions" не влияет на создание ссылок (Alt+F6). В этом случае
-права всегда копируются.
+ Bit 2    Bit 5    0 - ^<wrap>Defined by bits 0 и 1 / 3 и 4 of this parameter
```
Comment (alabuzhev) [inline far/FarEng.hlf.m4]: `и` -> `and`?

File: far/FarRus.hlf.m4 [hunk 3]
```diff
@@ -6221,41 +6221,56 @@ Manager префикс плагина к пути до объекта на па
...
- Аналогично, для операции копирования воздействие битов 3 и 4 зависит от состояния бита 5.
+ Опция #Права доступа# определяет права доступа, присваиваемые вновь
+создаваемым файлам и папкам, и может принимать одно из трёх значений:
 
- Примеры:
- 0x21 - ^<wrap>для операции перемещения опцию "Права доступа" выставлять всегда в "Копировать",
-для операции копирования выставлять опцию в значение по умолчанию и запоминать её значение до конца сеанса работы Far Manager.
- 0xС0 - ^<wrap>для перемещения запоминать значение опции до конца сеанса работы Far (при первом вызове диалога опция
-выставлена в значение по умолчанию), для операции копирования опцию "Права доступа" выставлять всегда в "Копировать".
+ #По умолчанию# - ^<wrap>Права доступа останавливаются файловой системой;
```
Comment (alabuzhev) [inline far/FarRus.hlf.m4]: `останавливаются` - `устанавливаются`? 
Same about FS.

File: far/FarRus.hlf.m4 [hunk 4]
```diff
@@ -6221,41 +6221,56 @@ Manager префикс плагина к пути до объекта на па
...
+
+ Значение по умолчанию: 0 (при открытии диалога опция #Права доступа#
+всегда устанавливается в значение #По умолчанию#; значения, задаваемые
+пользователем, не запоминаются).
+
+ Примеры:
 
- Параметр "System.CopySecurityOptions" не влияет на создание ссылок (Alt+F6). В этом случае
-права всегда копируются.
+ #0b100'001# - ^<wrap>В диалоге переноса опция #Права доступа# всегда
```
Comment (alabuzhev) [inline far/FarRus.hlf.m4]: Same about binary notation.

File: far/FarEng.hlf.m4 [hunk 5]
```diff
@@ -6121,43 +6121,56 @@ files, like #Temporary panel# does.
 
 @System.CopySecurityOptions
```
Comment (alabuzhev) [inline far/FarEng.hlf.m4]: Original English translation for reference:
https://github.com/FarGroup/FarManager/blob/release/2.0/docs/ENG/TechInfo.txt#L539-L580

**Comment by alabuzhev** [comment]
I have a feeling that it would be easier to create a UI dialog with 6 checkboxes than to make this TI understandable for laypeople.

File: far/FarEng.hlf.m4 [hunk 6]
```diff
@@ -6121,43 +6121,56 @@ files, like #Temporary panel# does.
...
- Аналогично, для операции копирования воздействие битов 3 и 4 зависит от состояния бита 5.
+ The #Access rights# option specifies the access rights assigned
+to newly created files and folders and can be one of:
 
- Примеры:
- 0x21 - ^<wrap>для операции перемещения опцию "Права доступа" выставлять всегда в "Копировать",
-для операции копирования выставлять опцию в значение по умолчанию и запоминать её значение до конца сеанса работы Far Manager.
- 0xС0 - ^<wrap>для перемещения запоминать значение опции до конца сеанса работы Far (при первом вызове диалога опция
-выставлена в значение по умолчанию), для операции копирования опцию "Права доступа" выставлять всегда в "Копировать".
+ #Default# - ^<wrap>Access rights are controlled by the file system;
```
[the remark this answers is not in the document]
Comment (MKadaner) [reply far/FarEng.hlf.m4]: Agreed, it's hard to tell exactly. OS is safer bet in this context. I changed the wording in `@CopyFiles` accordingly.

File: far/FarEng.hlf.m4 (hunk 1 above)
Comment (MKadaner) [reply far/FarEng.hlf.m4]: See if you like how I put it.

File: far/FarEng.hlf.m4 (hunk 2 above)
Comment (MKadaner) [reply far/FarEng.hlf.m4]: Ouch! Fixed.

File: far/FarRus.hlf.m4 (hunk 3 above)
Comment (MKadaner) [reply far/FarRus.hlf.m4]: Good catch, thank you. Fixed both.

File: far/FarRus.hlf.m4 (hunk 4 above)
Comment (MKadaner) [reply far/FarRus.hlf.m4]: Fixed.

File: far/FarEng.hlf.m4 (hunk 5 above)
Comment (MKadaner) [reply far/FarEng.hlf.m4]: Thanks, I've seen it. It is difficult to understand. I liberally reworked both English and Russian versions.

**Comment by MKadaner** [comment]
> I have a feeling that it would be easier to create a UI dialog with 6 checkboxes than to make this TI understandable for laypeople.

I feel this parameter is over-engineered as it is; creating a new dialog for it... Well, dunno. Other controls in the Copy / Move dialog (and in other dialogs, e.g., Search) are always set to their respective defaults on open, and nobody complains. Or do they? Why is this parameter so special?

Granted, I had hard time trying to understand what the parameter is doing exactly and resorted to reading the source code. The users are unlikely to have such luxury. This is why I choose to rework the description. As usual, I ran the result by my wife, and she said that while it required an effort, she certainly could make sense of it.

**Comment by MKadaner** [comment]
Please let me know if you are OK with this. I'll prepare final commit.

**Comment by MKadaner** [comment]
Thank you. Please merge.

File: far/changelog_eng [hunk 7]
```diff
@@ -3,6 +3,10 @@ This file is a translation of the main russian changelog and is provided by volu
 It might not always be as up to date as the main changelog.
 =======================================================================================
 
+MZK 07.03.2020 12:45:31 -0800
+
+1. Переработал страницу справки @System.CopySecurityOptions, английскую и русскую.
```
Comment (alabuzhev) [inline far/changelog_eng]: This probably belongs to the other changelog (and vice versa)?

Comment (MKadaner) [reply far/changelog_eng]: Sorry for the mess. Let's try again.

**Comment by alabuzhev** [comment]
Thanks!

**Pull request merged**